docs/user-guide/kubernetes.md

```diff
@@ -112,66 +120,147 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        args:
+        - --web
+        - --web.address=:8081
+        - --kubernetes
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: traefik-ingress-service
+spec:
+  selector:
+    k8s-app: traefik-ingress-lb
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 80
+    - protocol: TCP
+      port: 8081
```

Comment:
I think we should just use 8080, which happens to be the default port for Traefik's administration interface.